birdhouse/components/cowbird/user_workspace_readme.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6eb979d5-fc6f-456b-9931-6209958f4bae",
+   "metadata": {},
+   "source": [
+    "# README\n",
+    "\n",
+    "This is the user workspace for the JupyterLab environment.\n",
+    "\n",
+    "## Home Directory Layout\n",
+    "\n",
+    "```\n",
+    "notebook_dir\n",
+    "├── README.ipynb\n",
+    "├── public\n",
+    "├── tutorial-notebooks\n",
+    "└── writable-workspace\n",
+    "    ├── notebooks\n",
+    "    ├── shapefile_datastore\n",
+    "    └── wps_outputs \n",
+    "```\n",
+    "\n",
+    "Here is a description of the different directories found in the workspace :\n",
+    "\n",
+    "## public\n",
+    "\n",
+    "This directory contains any files that are shared publicly between the users.\n",
+    "This contains, for example, the different WPS outputs files that are not specific to any user.\n",
+    "Note that files shared here are read-only.\n",
+    "\n",
+    "## tutorial-notebooks\n",
+    "\n",
+    "This directory contains different notebooks used to showcase different usages of the birdhouse services and shares\n",
+    "step-by-step instructions of different use cases.\n",
+    "These notebooks can be executed but cannot be saved as they are read-only.\n",
+    "\n",
+    "## writable-workspace\n",
+    "\n",
+    "This directory contains different subdirectories that are customized and specialized to the user.\n",
+    "Note that this space is generally writable for the user, so they can create their own files in this directory as desired.\n",
+    "\n",
+    "#### notebooks\n",
+    "\n",
+    "This subdirectory is made to contain the different notebooks created by the user.\n",
+    "\n",
+    "#### shapefile_datastore\n",
+    "\n",
+    "This subdirectory is made to contain different files of the shapefile format for Geoserver. Adding new shapefiles to\n",
+    "this directory will also automatically publish the shapefile to the user's Geoserver workspace.\n",
+    "\n",
+    "#### wps_outputs\n",
+    "\n",
+    "This read-only directory contains the different WPS outputs user files. Note that this directory might not exist if the\n",
```

Comment:
Oh, now we see `wps_outputs/` dir is read-only.  Why nest it under `writable-workspace/` then?